platform/linux-generic/include/odp_pktio_ops_subsystem.h

```diff
@@ -87,12 +87,8 @@ typedef ODP_MODULE_CLASS(pktio_ops) {
 	odp_api_proto(pktio_ops, print) print;
 } pktio_ops_module_t;
 
-/* Maximum size of pktio specific ops data.*/
-#define ODP_PKTIO_ODPS_DATA_MAX_SIZE 80000
-
 /* Extract pktio ops data from pktio entry structure */
-#define odp_ops_data(_p, _mod) \
-	((pktio_ops_ ## _mod ## _data_t *)(uintptr_t)_p->s.ops_data)
+#define odp_ops_data(_p, _mod) _p->s.ops_data
 
 #define odp_ops_data_alloc(_p, _size)					\
```

Comment:
'_p' is not required as the macro is returning 'ops_data'. Makes the macro simple as well.

Similarly for odp_ops_data_free can just take 'ops_data' as input.

This will be inline with future plans to not expose 'pktio_entry_t' to the drivers.